packages/data_taster/docs/README.md

```diff
@@ -1,3 +1,75 @@
 # 🍭 Data Taster
 
-More info coming soon
+Database exporting tool.
+
+## Usage
+
+### `data_taster_export_tables.yml`
+
+With DataTaster, you can configure certain tables and rows to be pulled from the source_db. Look for (or create) a `data_taster_export_tables.yml` file.
```

Comment:
Where? At the app root? I wonder if we shouldn't move this into `db`

Reply:
in our monolith, these files live in `config/`. I'd like to avoid any kind of opinions on where people place these files, since apps can be configured differently and we don't have set standards for this here.